compose/src/main/java/com/bottlerocketstudios/compose/utils/map/ClusterMarkerTextView.kt

```diff
@@ -0,0 +1,39 @@
+package com.bottlerocketstudios.compose.utils.map
+
+import android.content.Context
+import android.graphics.Canvas
+import android.util.AttributeSet
+import android.widget.TextView
+
+class ClusterMarkerTextView : TextView {
```

Comment:
@Josh-Gebbeken If this implementation is based off of or inspired by another one, it would be nice to add a link to this class so that any future updates/changes could be reviewed and potentially integrated